components/PlantsPreview.js

```diff
@@ -1,12 +1,31 @@
 import Image from "next/image";
+import styled from "styled-components";
+import Link from "next/link";
 
-export default function PlantsPreview({ name, botanical_name, image }) {
+const StyledCard = styled.div`
+  width: 80vw;
+  height: auto;
+  border: 1px solid black;
+  background-color: #abd1c6;
+  border-radius: 20px;
+  &:hover {
+    box-shadow: 5px 5px 5px #f9bc60;
+  }
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #001e1d;
+`;
+
+export default function PlantsPreview({ name, botanical_name, image, id }) {
```

Comment:
It's usually a good practice to name components in singular form when they represent a single instance of an entity. Since `PlantPreview` is likely to represent a **single plant**, the singular form is more appropriate.

Please remember to rename the file as well

```suggestion
export default function PlantPreview({ name, botanical_name, image, id }) {
```